### PR TITLE
Fix actor struct offsets

### PR DESCRIPTION
**Pull request by Kelebek1**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

This was bothering me for a while.

### CODE REVIEW

**Review by jpburnett — APPROVED** [review]

**Review by tom-overton — APPROVED** [review]
Thanks for doing this!

**Review by EllipticEllipsis — APPROVED** [review]
Thanks for doing this

**Comment by EllipticEllipsis** [comment]
> Thanks for doing this!

Oh, didn't see this lol

File: src/overlays/actors/ovl_Obj_Bell/z_obj_bell.h [hunk 1]
```diff
@@ -11,7 +11,7 @@ typedef struct ObjBell {
     /* 0x1B4 */ ColliderSphere collider2;
     /* 0x20C */ s16 unk_20C; // bell rotation angle?
     /* 0x20E */ s16 unk_20E;
-    /* 0x20E */ s16 unk_20F;
+    /* 0x20F */ s16 unk_20F;
     /* 0x212 */ s16 unk_212;
```
Comment (engineer124) [inline src/overlays/actors/ovl_Obj_Bell/z_obj_bell.h:15]:
```suggestion
    /* 0x210 */ s16 unk_210;
    /* 0x212 */ s16 unk_212;
```

**Review by engineer124 — COMMENTED** [review]
Just one small comment. Nice work

**Review by AngheloAlf — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**